server/methods/core/shipping.js

```diff
@@ -4,6 +4,22 @@ import { Cart, Accounts } from "/lib/collections";
 import { Logger, Hooks } from "/server/api";
 import { Cart as CartSchema } from "/lib/collections/schemas";
 
+/**
+ * @file Methods for Shipping - methods typically used for checkout (shipping, taxes, etc).
+ * Run these methods using `Meteor.call()`.
+ *
+ *
+ * @namespace Methods/Shipping
+*/
+
+/**
+ * @name createShipmentQuotes
+ * @private
```

Comment:
Do we need a summary for this private method?

Reply:
Not required but good to have.